vision_datasets/common/data_manifest.py

```diff
@@ -742,6 +742,12 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
             images = [x for x in images_by_id.values()]
             return DatasetManifest(images, None, data_type)
 
+        if data_type == DatasetTypes.IMAGE_MATTING:
```

Comment:
We would need test for data_manfiest.

We should make sure manifest_dataset works with this as well.